src/ppspp_options.erl

```diff
@@ -294,14 +305,21 @@ use_default_options() ->
 use_default_options(Hex_String) when is_list(Hex_String) ->
     use_default_options(convert:hex_string_to_padded_binary(Hex_String));
 use_default_options(Swarm_id) when is_binary(Swarm_id) ->
-    {options,
-     orddict:from_list( [{swarm_id, Swarm_id},
-                         {chunk_addressing_method, chunking_32bit_chunks},
-                         {chunk_size, ?PPSPP_DEFAULT_CHUNK_SIZE},
-                         {content_integrity_check_method, merkle_hash_tree},
-                         {merkle_hash_tree_function, sha},
-                         {minimum_version, ?PPSPP_RFC_VERSION},
-                         {version, ?PPSPP_RFC_VERSION}])}.
+    #{swarm_id                       => Swarm_id,
+      chunk_addressing_method        => chunking_32bit_chunks,
+      chunk_size                     => ?PPSPP_DEFAULT_CHUNK_SIZE,
+      content_integrity_check_method => merkle_hash_tree,
+      merkle_hash_tree_function      => sha,
+      minimum_version                => ?PPSPP_RFC_VERSION,
+      version                        => ?PPSPP_RFC_VERSION}.
+    %% {options,
+    %%  orddict:from_list( [{swarm_id, Swarm_id},
+    %%                      {chunk_addressing_method, chunking_32bit_chunks},
+    %%                      {chunk_size, ?PPSPP_DEFAULT_CHUNK_SIZE},
+    %%                      {content_integrity_check_method, merkle_hash_tree},
+    %%                      {merkle_hash_tree_function, sha},
+    %%                      {minimum_version, ?PPSPP_RFC_VERSION},
+    %%                      {version, ?PPSPP_RFC_VERSION}])}.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% @doc pack an orddict of ppspp erlang terms into a binary PPSPP message segment
```

Comment:
Line 325 is too long: %% @doc pack an orddict of ppspp erlang terms into a binary PPSPP message segment.